src/components/LayoutTree.tsx

```diff
@@ -449,22 +484,51 @@ const LayoutTree = () => {
     [fontSize],
   )
 
+  /** Compare ranks between two thoughts. */
```

Comment:
This is a pure function and can/should be defined outside the context of the `LayoutTree` component.